demisto_sdk/commands/content_graph/interface/neo4j/queries/validations.py

```diff
@@ -40,14 +40,14 @@ def validate_unknown_content(
     """
     if include_optional:
         query = f"""// Returns USES relationships to content items not in the repository
-        MATCH (content_item_from{{deprecated: false}})-[r:{RelationshipType.USES}]->(n{{not_in_repository: true}})
-        WHERE NOT (content_item_from.is_test) {f'AND content_item_from.path in {file_paths}' if file_paths else ''}
+        MATCH (content_item_from{{deprecated: false, is_test: false}})-[r:{RelationshipType.USES}]->(n{{not_in_repository: true}})
```

Comment:
Following our conversation: this change affects only the direct USES relationships of test playbooks and test scripts, and in case of an issue the test playbook will fail anyway, plus it doesn't affect customers directly. So I'm ok with this change :+1: